portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ConnectionPropertyList.testcase

```diff
@@ -40,8 +40,13 @@ definition {
 	}
 
 	@description = "Story: LRAC-12019 | Automation ID: LRAC-12584 | Test Summary: Can paginate the list of properties"
+	@ignore = "true"
 	@priority = 3
 	test CanPaginatePropertyListInDXP {
+
+		// AC Bug: Properties are unable to be deleted
+		// AC Bug ticket: LRAC-13704
```

Comment:
Hey @joshitagaki, I think it would be good to create a ticket inside this [Epic](https://issues.liferay.com/browse/LRAC-13704) for us to enable these tests that are blocked by this. You can create a ticket and in the description say to search for LRAC-13704 to find the tests affected by the epic (I think it will be less work for you than having to list all the tests)

What do you think?

Reply:
I can add that. Thanks!